console-subscriber/src/builder.rs

```diff
@@ -158,3 +150,68 @@ impl Builder {
         self
     }
 }
+
+fn parse_duration(var_name: &str) -> Option<Duration> {
+    fn try_parse(s: &str) -> Result<Duration, Box<dyn std::error::Error>> {
+        let s = s.trim();
+
+        if let Some(s) = s.strip_suffix("ns") {
+            return Ok(Duration::from_nanos(s.trim().parse()?));
+        }
+
+        if let Some(s) = s.strip_suffix("us") {
+            return Ok(Duration::from_micros(s.trim().parse()?));
+        }
+
+        if let Some(s) = s.strip_suffix("ms") {
+            return Ok(Duration::from_millis(s.trim().parse()?));
+        }
+
+        if let Some(s) = s
+            .strip_suffix('s')
```

Comment:
Will this also match any units that the user ends with an `s`? Like `minutes`, `mins`, etc? Also, looking up, that'd mean `ns` would catch `2mins`?

Reply:
whoops, good catch --- this would then fail, because the rest of the string is not a valid integer/float. we should probably try to parse `s` last!